x-pack/elastic-agent/pkg/agent/application/fleet_server_bootstrap.go

```diff
@@ -7,6 +7,8 @@ package application
 import (
 	"context"
 
+	"go.elastic.co/apm"
```

Comment:
Interesting I didn't know about go.elastic.co, when the restructuring of the repository is complete we should take a look a it. @kvch @cmacknz @ruflin

Reply:
TBH, I am not a huge fan of vanity imports. It looks great, but it is just vanity... :)